File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-4.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int l=1;
        if(nums.length <1)
        {
            return nums.length;
        }
        //2,10,30,30,30,30
        for(int i=1;i<nums.length;i++)
        {
            if(nums[i] != nums[i-1])
            {
                nums[l]=nums[i];
                l++;
            }
        }
        return l;
    }
}

File: Data Structures & Algorithms/two-integer-sum/submission-4.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        hm = {}
        for i in range(0, len(nums)):
                target_check = target - nums[i]
                if target_check in hm:
                        return [hm.get(target_check),i]
                else:
                        hm[nums[i]]=i
        
        